src/app/container/dockerfile/dockerfile.component.ts

```diff
@@ -56,7 +56,7 @@ export class DockerfileComponent extends VersionSelector implements AfterViewChe
     }
   }
   ngAfterViewChecked() {
-    if (this.contentHighlighted && !this.nullContent) {
+    if (this.contentHighlighted && !this.nullContent && this.elementRef.nativeElement.querySelector('.highlight')) {
```

Comment:
Not sure if this is a thing in typescript, but does `this.elementRef.nativeElement` need a nullcheck or anything similar?

Reply:
I'm currently checking if it's truthy, it previously wasn't needed but somehow it is now.